delta-app/src/main/java/io/cdap/delta/app/QueueingEventEmitter.java

```diff
@@ -75,12 +72,10 @@ public void emit(DMLEvent event) {
       return;
     }
 
-    sequenceNumber++;
     try {
-      consumer.applyDML(new Sequenced<>(event, sequenceNumber));
-    } catch (Exception e) {
-      // TODO: (CDAP-16251) retry
-      throw new RuntimeException(e);
+      eventQueue.put(new Sequenced<>(event, ++sequenceNumber));
+    } catch (InterruptedException e) {
+      // this means the worker is being stopped, so drop the change and let things stop
```

Comment:
Should we at least log a warning for this exception?

Reply:
The contract in our plugins is that events are emitted at least once, so this is a normal and acceptable thing to happen since it will be re-read when the pipeline is started again

Reply:
makes sense.